### ISSUE TITLE
logged timestamps are wrong

### ISSUE THREAD

**Issue by atheurer** [perftool-incubator/tool-sysstat#25]
looking at a recent metric-data-0.csv.xz, timestamps are incorrect:

3,1618272000000,1618272000000,0.010000
2,1618272000000,1618272000000,0.983400
5,1618272000000,1618272000000,0.003300
8,1618272000000,1618272000000,0.003300
16,1618272000000,1618272000000,0.996700
13,1618272000000,1618272000000,0.003300
29,1618272000000,1618272000000,0.003300
23,1618272000000,1618272000000,0.996700

They do not advance.  This is certainly due to something wrong when going to the new post-processing.  I don't think it's the log_samples(), but something else git messed up in sysstat-post-process

**Comment by k-rister** [comment]
I think you made a mistake.  You replaced lines like this:

```
my %kbpi_sample = ( 'value' => $kb_paged_in, 'end' => $hms_ms + $ymd_timestamp_ms );
```

with lines like this:

```
my %sample = ('end' => $ymd_timestamp_ms);
```

so you dropped the `$hms_ms`.

**Comment by atheurer** [comment]
Yes, I discovered that earlier today.  I also have some other problems that may be related to metrics.pm

**Comment by atheurer** [comment]
fixed by #26
